global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "6.0.301"
```

Comment:
I'd update this to specify `"rollForward": "patch"`, and change the version to `6.3.00` - similar to [this](https://github.com/dotnet/roslyn-analyzers/blob/58db3ef1529ce44aef5bd6324deab15f7581c27c/global.json#L17).  This will allow us to keep up to date on the latest security patches without needing to constantly modify this file.

Reply:
Thanks for the feedback, Nick. @yaqian256 let me know when you push updates to address Nick's feedback and Pete's comments and I'll work on getting this in.

Reply:
Updated as suggested. Thanks!